daemon/rpc_connect_test.go

```diff
@@ -165,39 +174,6 @@ func TestRpcConnect(t *testing.T) {
 			checker: &workingLoginChecker{vpnErr: errors.New("test error")},
 			resp:    internal.CodeTokenRenewError,
 		},
-		{
```

Comment:
Would be good at least one test for DIP expired to check

Reply:
Expiration checks are done on the `RenewingChecker` side, there are some tests there: https://github.com/NordSecurity/nordvpn-linux/pull/503/files#diff-b06746870e62f95017714121be6134cffdc4bd1d46dd21f8cae8c484e3e2829dR253